doc/devops/keeper-setup.md

```diff
@@ -98,36 +123,28 @@ hosts = ["all"]
 
 [mining]
 engine_signer = "<ACCOUNT_ADDRESS>"
-reseal_on_txs = "none"
-usd_per_tx = "0"
-gas_floor_target = "4700000"
-
-[network]
-nat = "extip:<PUBLIC_IP>"
-port = 30303
-node_key = "<ETHEREUM_NODE_KEY>"
-discovery = true
-
-[ipfs]
-enable = false
-
-[snapshots]
-disable_periodic = true
+force_sealing = true
+reseal_on_txs = "none" // Authorities reseal automatically. From https://bit.ly/2Jiw5SV
+gas_floor_target = "6666666"
+usd_per_tx = "0" // Allows for free transactions. From https://bit.ly/2Jiw5SV
 
 [account]
-password = ["/etc/ocean/password"]
+password = ["/etc/parity/password"]
 
-[secretstore]
-disable = true
+[footprint]
+tracing = "on"
+pruning = "archive"
+fat_db = "on"
 ```
 
 Where:
-- ACCOUNT_ADDRESS: This is the ethereum address where you want to mine. Please put here your authority account if you have one.
-- PUBLIC_IP: The external public IP of your node.
-- ETHEREUM_NODE_KEY: The Ethereum node address
-- The file `/etc/ocean/password` contains the account's password. You can change the location of this file, but take care that this location reffers to a location inside the parity container, so you have to refer to the container location and mount the host's file to that location.
 
-4. Get the chain file for the network you're joining and save it in `/etc/ocean/chain.json`. Here are some links to chain files:
+- ACCOUNT_ADDRESS: Please put here your authority account address if you have one. Example: `0xdeadbeefcafe0000000000000000000000000001`
+- PUBLIC_IP: The external public IP address of your node. Example: `52.1.94.55`
+- ETHEREUM_NODE_KEY: The Ethereum node address. Example: `ade848e26c30d600da8207d0f8960d6abf125d1ac432bd42b020a98e10812f36`
+- The file `/etc/parity/password` contains the account's password. You can change the location of this file, but take care that this location refers to a location inside the parity container, so you have to refer to the container location and mount the host's file to that location.
+
+4. Get the chain file for the network you're joining and save it in `/etc/parity/chain.json`. Here are some links to chain files:
 
    - [Nile chain file](https://github.com/oceanprotocol/atlantic/blob/master/networks/nile/chain.json)
```

Comment:
atlantic is private and will probably stay private. if this guide is for public this has to be put somewhere else.

Reply:
Yes that is true. I think that we will eventually have a third option here, for a production-network chain file, and that will have to be stored somewhere public.